### ISSUE TITLE
Update steam-web dependence

### ISSUE THREAD

**Issue by Miwwa** [liamcurry/passport-steam#15]
Today I have exception
```
SyntaxError: Unexpected token <
  at Object.parse (native)
  at IncomingMessage.<anonymous> (/srv/csgo/node_modules/passport-steam/node_modules/steam-web/lib/steam.js:210:44)
  at IncomingMessage.emit (events.js:129:20)
  at _stream_readable.js:908:16
  at process._tickCallback (node.js:355:11)
```
In steam-web v.0.2.4 this issue has been fixed, you need to update version of this package

**Comment by vanesyan** [comment]
cc @liamcurry

**Comment by liamcurry** [comment]
Fixed in 565fc17175. Thanks for the report and reminder!

**Comment by Cuel** [comment]
I'm still geting it (passport-steam 0.1.6, steam-web 0.2.4)

```
undefined:1
<html><head><title>Forbidden</title></head><body><h1>Forbidden</h1>Access is d
^
SyntaxError: Unexpected token <
    at Object.parse (native)
    at IncomingMessage.<anonymous> (<path>\node_modules\passport-stea
m\node_modules\steam-web\lib\steam.js:258:44)
    at IncomingMessage.emit (events.js:129:20)
    at _stream_readable.js:908:16
    at process._tickCallback (node.js:355:11)
```

Is this an issue with steam-web?
